canonical-web-and-design/juju.is
# Build the about hosted-olm page

## QA
- Go to http://0.0.0.0:8041/about/hosted-olm
- Check the copy matches the [copy doc](https://docs.google.com/document/d/1zmGR4hZyyTLuPc98XahtcM7Tlb35QYzXZ9dWRf50l8Y/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/99
